fix wording, misprints, punctuation, and formatting (#169)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-android/overview.md

#### For websites

> You can add domains (e.g., `google.com`) or subdomains (e.g., `*.google.com`) of websites to the *Exclusions* in three ways: enter them manually in the app, or right from the browser by clicking the *Share* button and selecting AdGuard VPN in the opened list below, or from built-in lists of services divided by categories.

Reviewer: чет длинное предложение, его бы списком

```suggestion
You can add a domain (e.g., `google.com`) or subdomain (e.g., `*.google.com`) to *Exclusions* in three ways: 

- Enter it manually in the app
- Add it directly from the browser by sharing the website with AdGuard VPN
- Add it from a built-in list of services
```
